### ISSUE TITLE
MixedRealityToolkit Menu not displayed

### ISSUE THREAD

**Issue by crhodes2** [Microsoft/MixedRealityToolkit-Unity#3859]
Whenever I import MRTK--any versions, not the latest one--the Mixed Reality Toolkit menu refuses to be displayed. Is this normal?
If it's not, then how can I make the menu visible when I import it?
This had been an issue for the past couple of days. The latest version does display the menu but I am trying to use an older version for my project as the newest version isn't working out for me.

**Comment by davidkline-ms** [comment]
@crhodes2, not seeing the Mixed Reality Toolkit menu is generally caused by a script error that needs to be resolved.  Do you see any errors in the status bar or Console window?

**Comment by crhodes2** [comment]
Hi David,
So the errors are plenty. About 261 errors.
CS0246 which have a lot of different "namespaces could not be found" and CS0155 which have a lot of different "no suitable methods found to override"

**Comment by SimonDarksideJ** [comment]
Can you replicate the issue in a brand new clear project?   Just to ensure it's not something else (and the MRTK) that is causing issues.

Also please specify exactly the process you are using for getting the MRTK in your project (Downloading the asset / downloading source or cloning the repo)

**Comment by crhodes2** [comment]
I did.
I created a new Unity project, then once the project gets loaded, I located the MRTK package in Windows Explorer, double-clicked on it and have Unity import the package.
Same error.

**Comment by SimonDarksideJ** [comment]
It's odd as I just replicated those exact same steps today for a new video I was recording and didn't have that issue :S 

I did the following:
* Created a new Unity project (2018.3.11)
* downloaded the RC1 foundation package
* imported it in to Unity
* Ignored the crap Unity errors about being unable to load unity assemblies
* Clicked Run

Didn't have any errors pop up, so I suspect it may be an issue with your installation, so would love to find out what?  I take it if you follow the above steps you're fine?

Make sure you check
* Unity is installed with the UWP / Standalone IL2CPP clients
* Visual Studio is installed with the C++  UWP components (both the 2015 / 2017 runtimes) 
* you have a fresh cup of very strong coffee / tea

**Comment by crhodes2** [comment]
I'm gonna try reinstalling both Unity and Visual Studio with the recommended features. But just a quick heads up, the MRTK version I am using in particular is 2017.2.1. I have tried any other versions as well and got similar errors. The latter update works but I'm not using the latest update because working around with it is foreign to me and I have to relearn through its features.

I will try the latest version again if all else fail.

**Comment by SimonDarksideJ** [comment]
That is the old HoloToolkit, which is why we all for a version number when reporting issues.

I'll retest that version today, but that is the older toolkit which is going into support only mode due to the upcoming release of the mrtk2

But being the last stable release, it also shouldn't have any issues.

**Comment by chrisfromwork** [comment]
Assigning to @Yoyozilla for the time being.

**Comment by crhodes2** [comment]
I explained it already that the new version gives me the same issue as well.

But as a matter of fact, I have decided to go ahead and try the latest version: HoloToolkit 2017.4.3.0 - Refresh, and its example.

This is what's happening now...

Either I install it by drag-and-drop into the Assets folder of my project, or I just double-click on the MRTK files themselves, it is the exact same scenario.

This is the latest error I'm getting now...

Assets\HoloToolkit-Examples\SpectatorView\Scripts\SpectatorViewNetworkManager.cs(4,25): error CS0234: The type or namespace name 'Preview' does not exist in the namespace 'HoloToolkit.Unity' (are you missing an assembly reference?)

Assets\HoloToolkit-Examples\SpectatorView\Scripts\SpectatorViewNetworkManager.cs(19,13): error CS0246: The type or namespace name 'SpectatorView' could not be found (are you missing a using directive or an assembly reference?)

I don't know when am I ever going to complete this project? I have been doing this for the past 3-4 weeks to NO avail!
Every time I attempt to install MRTK and MRTK-Examples on my project--even new projects--
Errors. Nothing but errors.
